nrich-form-configuration/README.md

```diff
@@ -56,26 +71,46 @@ searched:
 - `client.NotNull.invalid`
 - `NotNull.invalid`
 
-`ConstrainedPropertyValidatorConverterService` is service responsible for converting constraints in a format client can interpret. Default implementation
-is `DefaultConstrainedPropertyValidatorConverterService` but users can register their own by implementing `ConstrainedPropertyValidatorConverterService` interface.
+---
+
+## ConstrainedPropertyValidatorConverterService
+
+[`ConstrainedPropertyValidatorConverterService`][constrained-property-validator-converter-service-url] is responsible for converting constraints in a format the client can interpret.
+Default implementation is [`DefaultConstrainedPropertyValidatorConverterService`][default-constrained-property-validator-converter-service-url] but users can register their own by implementing
+[`ConstrainedPropertyValidatorConverterService`][constrained-property-validator-converter-service-url] interface.
+
+[`ConstrainedPropertyValidatorConverterService`][constrained-property-validator-converter-service-url] returns a list of
+[`ConstrainedPropertyClientValidatorConfiguration`][constrained-property-client-validator-configuration-url] for each constraint (a list since some constraints on server may translate to multiple
+constraints on the client).
+
+---
 
-`ConstrainedPropertyValidatorConverterService` returns a list of `ConstrainedPropertyClientValidatorConfiguration` for each constraint
-(a list since some constraints on server may translate to multiple constraints on the client).
+## FormConfigurationService
 
-`FormConfigurationService` is service that processes constraints defined on a class for form id list and returns a list of
-`FormConfiguration` instances holding client side constrained property configuration.
+[`FormConfigurationService`][form-configuration-service-url] processes constraints defined on a class for form id list and returns a list of [`FormConfiguration`][form-configuration-url] instances
+holding client-side constrained property configuration.
 
-`FormConfigurationController` is REST API exposed to client. It has a single POST method `fetch` mapped to `nrich/form/configuration/fetch` that accepts a request holding form id list in
-property `formIdList` and it returns a list of `FormConfiguration` instances.
+---
+
+## FormConfigurationController
+
+[`FormConfigurationController`][form-configuration-controller-url] is REST API exposed to the client.
```

Comment:
Maybe include here that controller base path is configurable (```@RequestMapping("${nrich.form-configuration.endpoint-path:nrich/form/configuration}")```)

Reply:
Done.